spec/javascript/e2e/ApplicationNewPage.e2e.js

```diff
@@ -27,8 +30,8 @@ describe('ApplicationNewPage', () => {
     await sharedSteps.goto(page, `/listings/${NON_LEASE_UP_LISTING_ID}/applications/new`)
 
     // Enter in required information
-    await page.type('#first_name', FIRST_NAME)
-    await page.type('#last_name', LAST_NAME)
+    await page.type('#first_name', LONG_FIRST_NAME)
+    await page.type('#last_name', LONG_LAST_NAME)
```

Comment:
what is being tested here? that the whole string can be typed in?

Reply:
`LONG_FIRST_NAME` is a string over the 40 char limit for the input. The test makes sure that even if we type in `LONG_FIRST_NAME` that `FIRST_NAME` (which is the truncated string which at the max limit) is the value that actually gets saved.

Reply:
Added a comment to clarify :)

Reply:
maybe it'd be clearer if FIRST_NAME and LAST_NAME were TRUNCATED_FIRST_NAME and TRUNCATED_LAST_NAME

Reply:
done